app/assets/javascripts/settings/ldap-conn-test.js

```diff
@@ -0,0 +1,78 @@
+$(function () {
+  function NoneEmpty (arr) {
+    for (var i = 0; i < arr.length; i++) {
+      if (arr[i] === '') return false
+    }
+    return true
+  };
+
+  function RenderValidity (msg, isValid) {
```

Comment:
In javascript we usually write names in lower camel case.

Reply:
Yep, my mistake.  Fixed now.